helm/templates/service_clusterip.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
   labels:
     app: {{ .Release.Name }}
     metricsMonitor: {{ (.Values.service.http.metricsMonitor.enable | quote ) | default "true" }}
+  annotations:
```

Comment:
We should make this configurable, like:
 { if (((.Values.service.http).metricsMonitor).enable) -}}

Reply:
done